evap/evaluation/templates/base.html

```diff
@@ -185,6 +202,40 @@
             };
         </script>
 
+        <script type="module">
+            // Notebook logic
+
+            import { getCookie, setCookie} from "{% static 'js/utils.js' %}";
+            if (getCookie('evap_notebook_open') == "true") {
+                new bootstrap.Collapse(document.querySelector('#notebook'));
+                new bootstrap.Collapse(document.querySelector('#notebookButton'));  
+                onShowNotebook();
+            };
+
+            // make evap go away for notebook
+            $('#notebook').on('show.bs.collapse', function () {
+                onShowNotebook();
+                setCookie("evap_notebook_open", "true");
+            });
+
+            $('#notebook').on('hidden.bs.collapse', function () {
+                onHideNotebook()
+                setCookie("evap_notebook_open", "false");
+            });
+
+            function onShowNotebook()
+            {
```

Comment:
This is not our codestyle, please format like this:
```suggestion
            function onShowNotebook() {
```